master/pkg/schemas/expconf/experiment_config.go

```diff
@@ -81,6 +81,14 @@ func (e *ExperimentConfigV0) Scan(src interface{}) error {
 	return nil
 }
 
+// AsHistorical converts a current ExperimentConfig to a (limited capacity) HistoricalConfig.
+func (e ExperimentConfig) AsHistorical() HistoricalConfig {
+	return HistoricalConfig{
+		checkpointStorage: e.CheckpointStorage(),
+		bindMounts:        e.BindMounts(),
```

Comment:
no envvars?

Reply:
Just found/fixed that as part of the `Historical` -> `Legacy` rename.